buildSrc/src/testKit/jdk-download/build.gradle

```diff
@@ -1,12 +1,21 @@
-project.gradle.projectsEvaluated {
+allprojects{ p ->
```

Comment:
Is this file even used anymore since we've removed the old testkit tests?

Reply:
nope. cleaning up